src/content/docs/get-started/requirements.mdx

```diff
@@ -30,6 +30,14 @@ The following links provide the requirements and installation procedures for you
 
 <Task>
 
+### Storefront Compatibility package (optional)
+
+The [Storefront Compatibility package](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/launch-checklist/storefront-compatibility/) extends the GraphQL schema provided with Adobe Commerce. It provides several new mutations and adds missing fields that are needed to fully implement your dropins.
```

Comment:
The links returns 404

Reply:
@dshevtsov, Kevin created a PR to revert this (#50) but it hasn't been merged yet.

I would disregard the Jenkins prod build.